src/sensors.c

```diff
@@ -155,7 +160,9 @@ void init_sensors(void)
   mpu6050_register_interrupt_cb(&imu_ISR);
 
   uint16_t acc1G;
-  mpu6050_init(true, &acc1G, &gyro_scale, _params.values[PARAM_BOARD_REVISION]);
+  float gyro_scale_to_urad;
+  mpu6050_init(true, &acc1G, &gyro_scale_to_urad, _params.values[PARAM_BOARD_REVISION]);
+  gyro_scale = 1000000.0*gyro_scale_to_urad;
```

Comment:
Perhaps I'm just not understanding what `gyro_scale_to_urad` is doing. But if `raw * gyro_scale_to_urad` does actually give you a value in units of urad, then if the vehicle is rotating at a rate of 1 rad/s, `x = raw * gyro_scale_to_urad` would return a value of `x = 1e6 urad/s (= 1 rad/s)` rather than `x = 1e-6 urad (= 1e-12 rad)`. In that case we'd want to divide: `x / 1e6 = (1e6) / 1e6 = 1`. In other words, you need one million urad to equal one rad so `x` will be big; to get back to rad you have to _divide_ by one million.

?